applications/notifications/server/notifications/services/notification/backends/email_backend.py

```diff
@@ -0,0 +1,45 @@
+import smtplib
+
+from cloudharness import log as logger
+from cloudharness.utils.config import CloudharnessConfig as conf
+from cloudharness.utils.secrets import get_secret, SecretNotFound
+from email.message import EmailMessage
+from notifications.services.notification.backends.base_backend import NotificationBaseBackend
+
+
+DOMAIN = conf.get_configuration()["domain"]
+EMAIL_HOST = conf.get_configuration()["smtp"]["host"]
+EMAIL_PORT = conf.get_configuration()["smtp"]["port"]
+EMAIL_TLS  = conf.get_configuration()["smtp"].get("use_tls")
+try:
+    EMAIL_USER = get_secret('email-user')
```

Comment:
correct, but the allvalues.yaml currently isn't refreshed because it uses subPath see the note here https://kubernetes.io/docs/tasks/configure-pod-container/configure-pod-configmap/#mounted-configmaps-are-updated-automatically 

add. I changed the mounting of the allvalues.yaml in order to support auto refresh (sync)